services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from django.db.models import QuerySet
+from db.models import MovieSession
+from services.cinema_hall import get_hall_by_id
+from services.movie import get_movie_by_id
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=get_movie_by_id(movie_id),
+        cinema_hall=get_hall_by_id(cinema_hall_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if not session_date:
+        return MovieSession.objects.all()
+    year, month, day = session_date.split("-")
```

Comment:
There is no need to make split here, pass string object to filtration as is to the Django ORM